salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-examples.adoc

```diff
@@ -0,0 +1,300 @@
+== Before You Begin
+
+* Java 8 or 11
+* Anypoint Studio 7.5 and later
+* Mule runtime engine (Mule) 4.3.0 and later
+* DataWeave
+//not sure about this one:
+*OAth credentials for Salesforce Marketing Cloud  
+
+== Configure a Connection
+
+To secure connections, you must specify the connection field
+values. To do this:
+
+* <<create-config-file, Create a configuration file for a connection.>>
+* <<configure-conn, Configure the connection global elements.>>
+* <<configure-global, Configure a global element for the properties file.>>
+
+[[create-config-file]]
+=== Create a Configuration File for a Connection
+
+Create a configuration file that includes properties for a connection:
+
+. Create a file named `mule-app.properties` in the `/src/main/resources/` folder.
+. In the `mule-app.properties` file, create a set of properties for the connection,
+similar to the ones that follow, replacing the bracketed text (including the brackets)
+with the correct values for your configuration:
++
+----
+config-with-oauth.clientId=<clientId used to initialize the connection>
+config-with-oauth.clientSecret=<clientSecret used to authenticate the user>
+config-with-oauth.soapEndpoint=<SOAP Endpoint address>
+config-with-oauth.authEndpoint=<Authentication Endpoint>
+----
++
+This may vary depending on the selected connection configuration.
+
+For more information about creating a properties file, refer to xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+[[configure-conn]]
+=== Configure the Connection Global Elements
+
+Configure global elements for connection:
+
+. Create a new Mule project.
+. In the *Mule Palette* view, click *Search in Exchange* and enter `Salesforce Marketing Cloud`.
+. Add *Salesforce Marketing Cloud Connector* to the *Selected modules* section and click *Finish*.
+. Click the *Global Elements* tab and click *Create*.
+. Select *Connector Configuration > Salesforce Marketing Cloud* and click *OK*.
+. Enter the values to configure either Basic Authentication or OAuth Client Credentials.
+. Click the *Test Connection* button to ensure there is connectivity with the Salesforce Marketing Cloud API. A successful message should pop up.
+. Click *OK*.
+. Open the HTTPS Listener config in *Global Element Configuration*.
+. Click the *TLS* tab and select TLS Configuration > Edit inline.
+. Specify the *Key Store Configuration* with the generated keystore details to enable HTTPS on this configuration.
+. Click *OK*.
+
+[[configure-global]]
+=== Configure a Global Element for the Properties File
+
+Configure a global element for the `mule-app.properties` file so that Mule knows
+where to find it:
+
+. Click the *Global Elements* tab and click *Create*.
+. In the *Choose Global Type* dialog, select *Configuration properties* and click *OK*.
+. In the *File* field, enter `mule.app.properties`.
+. Click *OK*.
+
+
+[[create-list]]
+== Create a List
+
+This Mule flow creates a list for storing subscribers.
```

Comment:
May want to clean up and add this info in this intro sentence too (got this from the demo folder)

For List type, the default value is Public, meaning the list is visible to all subscribers. After creating the list, be sure you note the list id because this will be useful for other operations